lib/src/common/repos_service.dart

```diff
@@ -208,86 +247,192 @@ class RepositoriesService extends Service {
         (json) => Collaborator.fromJson(json),
       );
 
-  Future<bool> isCollaborator(RepositorySlug slug, String user) {
-    return _github
-        .request("GET", "/repos/${slug.fullName}/collaborators/$user")
-        .then((response) {
-      return response.statusCode == 204;
-    });
+  Future<void> isCollaborator(RepositorySlug slug, String user) async {
+    return _github.request(
+      "GET",
+      "/repos/${slug.fullName}/collaborators/$user",
+      statusCode: 204,
+    );
   }
 
-  Future<bool> addCollaborator(RepositorySlug slug, String user) {
-    return _github
-        .request("PUT", "/repos/${slug.fullName}/collaborators/$user")
-        .then((response) {
-      return response.statusCode == 204;
-    });
+  Future<void> addCollaborator(RepositorySlug slug, String user) async {
+    return _github.request(
+      "PUT",
+      "/repos/${slug.fullName}/collaborators/$user",
+      statusCode: 204,
+    );
+  }
+
+  Future<void> removeCollaborator(RepositorySlug slug, String user) async {
+    return _github.request(
+      "DELETE",
+      "/repos/${slug.fullName}/collaborators/$user",
+      statusCode: 204,
+    );
+  }
+
+  /// Returns a list of all comments for a specific commit.
+  ///
+  /// https://developer.github.com/v3/repos/comments/#list-commit-comments-for-a-repository
+  Stream<CommitComment> listComments(
+    RepositorySlug slug,
+    RepositoryCommit commit,
+  ) {
+    return PaginationHelper(_github)
+        .objects<Map<String, dynamic>, CommitComment>(
+      "GET",
+      "/repos/${slug.fullName}/commits/${commit.sha}/comments",
+      (i) => CommitComment.fromJSON(i),
+      statusCode: StatusCodes.OK,
+    );
   }
 
-  Future<bool> removeCollaborator(RepositorySlug slug, String user) {
-    return _github
-        .request("DELETE", "/repos/${slug.fullName}/collaborators/$user")
-        .then((response) {
-      return response.statusCode == 204;
+  /// Returns a list of all commit comments in a repository.
+  ///
+  /// https://developer.github.com/v3/repos/comments/#list-comments-for-a-single-commit
+  Stream<CommitComment> listCommitComments(RepositorySlug slug) {
+    return PaginationHelper(_github)
+        .objects<Map<String, dynamic>, CommitComment>(
+      "GET",
+      "repos/${slug.fullName}/comments",
```

Comment:
is this URL reversed with the one here? https://github.com/DirectMyFile/github.dart/pull/168/files#diff-7c401393bc787e1f6ada95787267561dR284